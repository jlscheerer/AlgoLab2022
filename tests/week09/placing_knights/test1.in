50
1
1 
2
1 1 
1 1 
2
0 1 
0 1 
2
0 1 
0 1 
2
0 1 
0 1 
3
1 1 1 
1 1 1 
1 1 1 
3
0 1 1 
1 1 0 
1 0 0 
3
0 1 1 
1 1 0 
1 0 0 
3
0 1 1 
1 1 0 
1 0 0 
3
0 1 0 
1 0 0 
1 0 0 
3
0 1 0 
1 0 0 
1 0 0 
3
0 1 0 
1 0 0 
1 0 0 
3
0 0 0 
1 0 0 
0 0 0 
3
0 0 0 
1 0 0 
0 0 0 
3
0 0 0 
0 0 0 
0 0 0 
5
0 1 1 1 1 
0 1 0 0 1 
1 1 1 1 1 
1 1 0 0 1 
1 1 1 1 1 
5
0 1 1 1 1 
0 1 0 0 1 
1 1 1 1 1 
1 1 0 0 1 
1 1 1 1 1 
5
0 1 1 1 1 
0 1 0 0 1 
1 1 1 1 1 
1 1 0 0 1 
1 1 1 1 1 
5
0 1 0 1 0 
0 1 0 0 0 
0 1 0 0 0 
1 0 0 0 1 
1 1 1 0 1 
5
0 1 0 1 0 
0 1 0 0 0 
0 1 0 0 0 
1 0 0 0 1 
1 1 1 0 1 
5
0 1 0 1 0 
0 1 0 0 0 
0 1 0 0 0 
1 0 0 0 1 
1 1 1 0 1 
5
0 0 0 1 0 
0 0 0 0 0 
0 0 0 0 0 
1 0 0 0 1 
1 0 0 0 0 
5
0 0 0 1 0 
0 0 0 0 0 
0 0 0 0 0 
1 0 0 0 1 
1 0 0 0 0 
6
1 1 1 1 1 0 
1 0 1 1 1 1 
1 1 1 1 1 1 
1 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 1 1 0 
6
0 1 1 1 1 0 
1 0 1 1 1 1 
1 1 1 1 1 1 
0 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 1 1 0 
6
0 1 1 1 1 0 
1 0 0 1 1 1 
1 1 1 1 1 0 
0 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 0 1 0 
6
0 1 1 1 1 0 
1 0 0 1 1 1 
0 0 1 1 1 0 
0 1 1 1 1 1 
1 1 1 1 1 0 
0 1 1 0 1 0 
6
0 1 0 1 0 0 
1 0 0 0 0 1 
0 0 0 1 0 0 
0 1 1 1 1 0 
1 1 1 1 1 0 
0 1 1 0 1 0 
6
0 1 0 1 0 0 
1 0 0 0 0 1 
0 0 0 1 0 0 
0 1 1 1 1 0 
0 1 1 1 1 0 
0 1 1 0 1 0 
6
0 1 0 1 0 0 
0 0 0 0 0 0 
0 0 0 1 0 0 
0 1 1 1 0 0 
0 0 0 1 0 0 
0 1 1 0 1 0 
6
0 0 0 1 0 0 
0 0 0 0 0 0 
0 0 0 1 0 0 
0 1 1 0 0 0 
0 0 0 1 0 0 
0 1 1 0 1 0 
6
0 0 0 0 0 0 
0 0 0 0 0 0 
0 0 0 0 0 0 
0 1 0 0 0 0 
0 0 0 0 0 0 
0 1 0 0 1 0 
6
1 1 1 1 1 0 
1 0 1 1 1 1 
1 1 1 1 1 1 
1 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 1 1 0 
6
0 1 1 1 1 0 
1 0 1 1 1 1 
1 1 1 1 1 1 
0 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 1 1 0 
6
0 1 1 1 1 0 
1 0 0 1 1 1 
1 1 1 1 1 0 
0 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 0 1 0 
6
0 1 1 1 1 0 
1 0 0 1 1 1 
0 0 1 1 1 0 
0 1 1 1 1 1 
1 1 1 1 1 0 
0 1 1 0 1 0 
6
0 1 0 1 0 0 
1 0 0 0 0 1 
0 0 0 1 0 0 
0 1 1 1 1 0 
1 1 1 1 1 0 
0 1 1 0 1 0 
6
0 1 0 1 0 0 
1 0 0 0 0 1 
0 0 0 1 0 0 
0 1 1 1 1 0 
0 1 1 1 1 0 
0 1 1 0 1 0 
6
0 1 0 1 0 0 
0 0 0 0 0 0 
0 0 0 1 0 0 
0 1 1 1 0 0 
0 0 0 1 0 0 
0 1 1 0 1 0 
6
0 0 0 1 0 0 
0 0 0 0 0 0 
0 0 0 1 0 0 
0 1 1 0 0 0 
0 0 0 1 0 0 
0 1 1 0 1 0 
6
0 0 0 0 0 0 
0 0 0 0 0 0 
0 0 0 0 0 0 
0 1 0 0 0 0 
0 0 0 0 0 0 
0 1 0 0 1 0 
6
1 1 1 1 1 0 
1 0 1 1 1 1 
1 1 1 1 1 1 
1 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 1 1 0 
6
0 1 1 1 1 0 
1 0 1 1 1 1 
1 1 1 1 1 1 
0 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 1 1 0 
6
0 1 1 1 1 0 
1 0 0 1 1 1 
1 1 1 1 1 0 
0 1 1 1 1 1 
1 1 1 1 1 0 
1 1 1 0 1 0 
6
0 1 1 1 1 0 
1 0 0 1 1 1 
0 0 1 1 1 0 
0 1 1 1 1 1 
1 1 1 1 1 0 
0 1 1 0 1 0 
6
0 1 1 0 1 1 
1 0 1 1 0 0 
1 0 0 0 1 0 
1 1 0 0 0 0 
1 0 1 0 0 0 
0 0 1 0 0 1 
6
0 0 1 0 1 1 
1 0 0 0 0 0 
0 0 0 0 1 0 
1 1 0 0 0 0 
1 0 0 0 0 0 
0 0 0 0 0 1 
6
0 0 0 0 1 1 
0 0 0 0 0 0 
0 0 0 0 1 0 
1 1 0 0 0 0 
1 0 0 0 0 0 
0 0 0 0 0 1 
6
0 0 0 0 1 1 
0 0 0 0 0 0 
0 0 0 0 0 0 
0 1 0 0 0 0 
0 0 0 0 0 0 
0 0 0 0 0 0 
6
0 0 0 0 0 0 
0 0 0 0 0 0 
0 0 0 0 0 0 
0 0 0 0 0 0 
0 0 0 0 0 0 
0 0 0 0 0 0 
