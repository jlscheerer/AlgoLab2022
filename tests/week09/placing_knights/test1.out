1
4
2
2
2
5
3
3
3
2
2
2
1
1
0
10
10
10
7
7
7
4
4
16
15
14
13
10
10
7
6
3
16
15
14
13
10
10
7
6
3
16
15
14
13
9
7
6
3
0
